### DISCUSSION TITLE
Actively Developed?

### DISCUSSION THREAD

**Discussion by File4mat** [soundshed/soundshed-app#72] [General]
I was curious if this app has a roadmap or is still being worked on? Ive come across some bugs and general usability issues. Before reporting everything and making suggestions I thought I would see if its worth the time investment.

While on the topic I would be happy to donate my time creating / editing graphics.

**Comment by webprofusion-chrisc** [comment]
I don't have any current plans for it, but if it completely stopped working I'd probably fix it. Based on the release download stats, it probably has a couple of thousand occasional users.

**Reply by webprofusion-chrisc** [reply]
The most recent thing I thought about doing was showing the FX chain as a 3D pedalboard, but while it looked cool it was more work than I really wanted to do and with not much benefit. https://github.com/soundshed/soundshed-app/discussions/66

Really the way to get the tones parts of the app more useful is to support other brands of amps/fx and provide translation between different preset formats (e.g. convert spark tones to line 6 pod go and vice versa etc). A lot of amps (katana etc) still don't have online enabled tone databases. This was one of the original aims.

**Reply by File4mat** [reply]
Thanks for the reply. I understand where you are coming from.

**Comment by gfm2000** [comment]
G’day Christopher

First off, you’ve done an amazing job with the app.

I am visually impaired and use a screenreader for everything. Most things read okay, some work great and some not at all. Soundshed runs rings around the Spark IOS app, which is pretty much useless to someone who can’t make use of the visuals.

If and when you ever feel like making a few tweaks to the app, it’d be great if you could label the controls so my screenreader can speak them aloud. Most of the time, it’s as simple as just captioning the control (most, not all!).

Anyway, it’d be appreciated if you could. If not, the app is still useful to me, and I, like the others who use it, appreciate all you’ve done.

 

Regards

Gordon
